<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>location send</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>141</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>bat</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_err">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_bat">
       <property name="text">
        <string>54.2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>err</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_x">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mx">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_y">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_my">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_th">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+th</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mth">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-th</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="robotx">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="roboty">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="robotth">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Ready">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ready</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Moving">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>moving</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
